polaris/network/layers.py

```diff
@@ -0,0 +1,67 @@
+"""Groups the defined basic layer types used for the project"""
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class SqueezeExcitation(nn.Module):
+    """Construct a squeeze excitation block for use in a residual block.
+
+    This uses the squeeze excitation block as proposed by `Hu et al. 2017`_.
+
+    .. _Hu et al. 2017:
+        https://arxiv.org/abs/1709.01507
+
+    """
+
+    def __init__(self, channels, ratio):
+        super().__init__()
+        self.dense_linear_1 = nn.Linear(channels, channels // ratio)
+        self.relu = nn.ReLU(inplace=True)
```

Comment:
Overall looks good but I still don't get why you put relu in init? you can just call `x = F.relu(x)` inside `forward`

Reply:
Thanks for reminding me, somehow I forgot to move it to the forward method.